### PR TITLE
Read compressed LHE weights

### PR DESCRIPTION
**Pull request by mseidel42**
Read compressed LHE weights through Pythia8

### CODE REVIEW

**Comment by perrozzi** [comment]
please test

**Comment by intrepid42** [comment]
File system hickup?

**Comment by perrozzi** [comment]
please test
(let's retry)

File: GeneratorInterface/Pythia8Interface/test/pythia8ex12_cfg.py [hunk 1]
```diff
@@ -30,7 +26,11 @@
               'SpaceShower:pTmaxMatch = 2',
               'TimeShower:pTmaxMatch = 2',
         ),
-        pythia8_example12 = cms.vstring('POWHEGres:calcScales = on'),
+        pythia8_example12 = cms.vstring(
+              'POWHEGres:calcScales = on',
+              'Beams:frameType = 4',
+              'Beams:LHEF = powheg-b_bbar_4l.lhe',
```
Comment (perrozzi) [inline GeneratorInterface/Pythia8Interface/test/pythia8ex12_cfg.py]: naive question: this is meant to be changed or is a leftover?

**Comment by intrepid42** [comment]
It is on purpose, so that the LHE file is read by Pythia and not by the CMS LHEReader

File: GeneratorInterface/Pythia8Interface/test/pythia8ex12_cfg.py (hunk 1 above)
Comment (intrepid42) [inline GeneratorInterface/Pythia8Interface/test/pythia8ex12_cfg.py]: It is on purpose, so that the LHE file is read by Pythia, not by the CMSSW LHEReader

**Comment by perrozzi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**